.circleci/config.yml

```diff
@@ -196,13 +196,9 @@ workflows:
           requires:
             - build-test-ova
           filters:
-            branches:
-              only:
-                - master
-                - feature/deploy-releases
             tags:
               only:
-                - /^v.*/
+                - /^v\d+\.\d+\.\d+-\d{8}$/
```

Comment:
I think `^v.*` was good enough, shouldn't be that explicit.
But OK ;)

Reply:
Just thought it might be a good idea to ensure we don't deploy if the tag is not a valid release tag. :)